dap_aria_mapping/pipeline/make_topic_name_assignments.py

```diff
@@ -0,0 +1,432 @@
+            ),
+        )
+
+        logger.info("Building dictionary - journal to entities")
+        journal_entities = get_journal_entities(oa_works, oa_entities)
+
+    if "chatgpt" in args.name_type:
+
+        OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
+        chatbots = {
+            "chatbot1": Chatbot(
+                config={
+                    "session_token": "eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..heSMPbaDFqCLUD6K.GAPPWlYIay-_b0GgIMYb6Ui5rAGxpSa4BOevkxNvilLSq95iDgB8cY0VUy1QqWRUPcCBQepufAsDAn3CvrbVTpfFWtB8kCaLQwL2MO_cXlZPowzy3YHcHz1_12vkreggkL9IVKdNaMlOlne9AVPZNLAhxDhu6Nerlw_ZcCEbeglhonDWnDTPSDbf_HzZT2SwhhKUFWg3iPlj7GHX6mX_UmaAv2nNOs2xTX34rcfOFjYHCO48WToDipy8Z2qTJdxnwJgnRK4V_uH-ucXKFc0yE4H612P064vsMpmEEaVwsqM1EZIFrcTLxmmIqg3MF2w77jx5XHBN7mMMPdVY9aV-h8E0nqFJLkmUJhaWWZIa0vlnOaEHLmllMgDLiHN3ur5-75gLFbHxuSgcOkrLMso4BCrSx22pBkyCLUMYu3D5N9q2zD3vn_nSVE52st1Lg5P64wfKNJIJk1GH1U1BmiALyatrVpHOMHBeNZ5ZpZRKEvZSXQILts-Ej1wYLZ35greMUy72oZTsuLTMLR2a_7py4-8pm2TvKJgWxgnBzbtfcli5-CZhYsjnV_AlyVh7KbZrMhw8_QuCHB5zP7xsUmq0tVi3Em7obGP4VRVJaIN4sxA6ZV3gqdFAmLtBITBSJh3jB74Mdl1XTXSz2Cvu33pvF8qlKxZVXcgGvb1HXuqfQQlAnpwNj_CthV0PBERhm7dR5nTaiRc9i9MGdwBfpga32zEVR_ZHxkp5DFs3UuS0IwN4s6HRMDy5bM8ytKupVZ6_Q4OVoZKOce40bK4OsVkHGzsivjhICNnjMJKh8Oqarf1vgP2xqWKF9rRoDcP_EEsvbjM_JsWxbvjOaEVDcqEwVOJ_suPOtEy2s96yi_ZRMOPLp52TtNdzev5_SZvznlJCnzbLM0JDV9RNarsHpcHzBu84IY-ydzcXRlz80SBWHkSNcTd9tMcTkNQG4f3yJIK-ufhfoWXme4fdBHhVICRBoe868hGYvkojRPRA3PXOisHOeHp7ERwgWQdNOErBBvly3M0B87QZT4ceGZKLDW1REJRbsXTpn0lb8ZI_OHVfvvizIHA8KSQyUE2euCtzdT2J5UJZUiO90ksui5QJKcUf2HGSL9WYPsTp8pOJNMD7qbmCAbp1u8dLhSxlvXmNsoo2EGLSGnPvPv33HWC3M7xnS7Nb-pC3RqBMqFPUI7vPj9EyE9HYIs2D72M0o-_AJZzEGvzcAJAo9nG59v8aFzWOMuWwrglNBAUh1wJt_mX3AVKwEFU_cQdTVAmXb4thUUFcbZ-_CI2b67XtZxi9N5t9BnIxhozbF2tC9gIk04scLdHTa5nJXsAOyjeGQwghIgC8CEzbjP7cOEG0ukWr0ykSjY3pQlLyDWJK37C57RFE9zQ4hgym__8INJ3ytS59qEDjnj7r8ZPh-_HlRFUFBJGrchndTHqwpB19q_ycb2p5-aiRXdYLu-hIEqxpNQbyPzw5Q32wUwqRw3SscZcstpDJfNMN_tLOABEZwheL14Cjm3noZS9S97mlyJeb0wwHR3gfuAcdsiHWN95Ri9Urlh13M-iRS-HiYQ8UJPapB4jkWwFpJCPCpqw7Pxq50KDLSJUMtQVGFd6_OYNG1vOuXj929M-YEjjluQQY8diFv-_i3bUTuVD8jnhvi-zgjMLM9BDgTsMnDwVFVb0U1o284paM_fJVxN7SxKe7Rl7hvqwelYXYi2GOwL10YbL0v3jjj5gqKf_6qNXIKNH1tVWUYQRvFlu8lkUF3usBZqBh1RtIBHSemV1a0XyrX5drUVzsvXN4NjR6EdmVnEOkAZO4VE5KLXkajf8KTm_vWOwxpnyKz-rZY-Mq7wr3X5k9TYgsXqaPceEyIiQBKLeWQYeDqRnQqFOL0GkDLwKdf9tb7IbdltWVef3FL28tbAUk_sMpRdpVyDQUGMLwzKLuDgw9cOtgUJ4PBtsCAgFaGxJF3JZQb2eE1vnCfQr08VD_1oUc3mDdjnGTsFWmLObcqpEoG141jhuoiw64EsCSVwq6SeWDIqUlr_XGh2DC-yTXF0GAkDv79Iq-xw6752d95FwxUKMT159aRDx1MR2HYtOZ2GyrpavFNWiaD-mbzKnRiHEd5RWLX0jeIpNCRWw6cINJ8gjU7gL0_wrWIexDAu7PwZPhAT5blSNyQl4JB1UfBOLlIyTgu2Cl_9R9a7rXC1tkmiGNNeDmkI6gkE1nH1wam5c0sr5Qnh5ad64OhAynkrKbk90bCUWGaRc3NWDsmCbKei0PU8xMRyAxle_CefeiskS1zMfZeMhrbXFfsuIRk3GHadGu0RWL00tvR7L-DYkHyFIxq-DGKZiV38oIH72u95k6ZXG2LUqSfp6IulYF0Ogo4w6-vbruBfTDFEvNQFkiHmmc0CDb-QiWlQ4fzc6sUj9HjcpMPXay_PeA5JZDOlu1zLUBsA-82CYEUQvX1PUHfE-vQuvr_r7edfeU6bvUOfy7mdpYuab8Zg3vT7XhCvIZpsMZYGPIVohjlIn1.lt2c4obxI6XeRfLV8BKH6g"
```

Comment:
maybe load these in configs in from a file so the long string isn't in the code?

Reply:
Good point!